example/all.yaml

```diff
@@ -173,6 +173,7 @@ linter:
     - unnecessary_getters_setters
     - unnecessary_lambdas
     - unnecessary_late
+    - unnecessary_library_directive
```

Comment:
Could we rename this to `unnecessary_library_directives`?  The plural seems more consistent w/ other similar lints.

Reply:
Meh...we have a lot of singular ones up there to ☝️ – since you can only have one `library` directive in a...`library` going simple seems better to me. Happy to discuss more – or curious about others opinions...